frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1044,6 +1045,27 @@ const ManageHostsPage = ({
         query: searchQuery,
         status,
         labelId,
+        currentTeam: teamIdForApi,
+        policyId,
+        policyResponse,
+        softwareId,
+        softwareTitleId,
+        softwareVersionId,
+        osName,
+        osVersionId,
+        osVersion,
+        // DisableFailingPolicies
+        // MacOSSettings
+        // MacOSDiskEncryptionFilter
+        // MDMBootstrapPackage
+        mdmId,
+        // MDMName
```

Comment:
same as other comments – if for future implementation, please communicate that

Reply:
@jacobshandling This is a draft PR, it's not assigned out for review yet and is still being worked on. The comments are for myself, I opened the draft already to be transparent with the team as this is a critical bug. I'll assign you when it's ready and I take it out of draft.

Reply:
I deleted the comments since they're popping up in my vs code